### PR TITLE
Move album field to beneath label field

### PR DESCRIPTION
**Pull request by akash1810**
Also renames it, within the ui, to photoshoot. This is after talking with Joanna and Josy.

# Image preview page
image

# Info panel on search
image

# Uploads page
image

### CODE REVIEW

File: kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html [hunk 1]
```diff
@@ -297,6 +288,17 @@
     </dl>
 </div>
 
+<div class="image-info">
+    <div class="image-info__group">
+        <dl class="image-info__wrap">
+            <dt class="metadata-line metadata-line__key">Photoshoot</dt>
+            <dd class="metadata-line__info">
+                <gr-album images="[ctrl.image]"></gr-album>
```
Comment (franziskas) [inline kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: is it worth changing the tag `<gr-album>` for consistency?

Comment (franziskas) [reply kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: See also https://github.com/guardian/grid/search?q=album&unscoped_q=album

Comment (akash1810) [reply kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: Yeah, that's definitely the correct thing to do. We'd need to also update the ES mappings, which I think should be fine...

Comment (akash1810) [reply kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: Done. Given the numbers, I think we can manually migrate any existing images with an `album`, moving them to `photoshoot`.

**Pull request merged**